arch/X86/X86Mapping.c

```diff
@@ -3467,13 +3467,16 @@ void op_addAvxBroadcast(MCInst *MI, x86_avx_bcast v)
 #ifndef CAPSTONE_DIET
 // map instruction to its characteristics
 typedef struct insn_op {
-	uint64_t eflags;	// how this instruction update EFLAGS
+	union{
+		uint64_t eflags;	// how this instruction update EFLAGS
+		uint64_t fpuflags;
```

Comment:
can you rename this field to fpu_flags, so it is consistent with other fields? (eflags is special, as it is associated with the EFLAGS register of x86)